bazel/external_crates.bzl

```diff
@@ -567,6 +569,12 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
                     "pem",
                 ],
             ),
+            "ic-agent-call-v3": crate.spec(
+                package = "ic-agent",
+                git = "https://github.com/dsharifi/agent-rs",
```

Comment:
Why can't we use dfinity's repo here (https://github.com/dfinity/agent-rs)?

Reply:
We don't want to merge the changes to agent-rs yet, and I do not have write access to create branches in the agent-rs repo.

Reply:
Also we do have precedents, e.g. the  git = "https://github.com/blind-oracle/cloudflare-rs.git",

Reply:
> I do not have write access to create branches in the agent-rs repo.

I just gave the [core](https://github.com/orgs/dfinity/teams/core) team Write access to dfinity/agent-rs. Everybody should be able to push code to that repo. We have branch protection in place to protect the `main` branch.

Reply:
Changed it to use a revision of dfinity's repo